src/cluster/cluster.cpp

```diff
@@ -36,15 +36,22 @@ Cluster::setup()
 	_max_fd = 0;
 	for (t_array::iterator it(_all_server_config.begin()); it != _all_server_config.end(); it++)
 	{
-		int	   fd;
-		Socket new_socket(AF_INET, it->get("port").get<double>(), SOCK_STREAM, 0, *it);
-		fd = new_socket.get_socket_id();
-		FD_SET(fd, &_master_fd_set);
-		_sockets.insert(std::make_pair(fd, new_socket));
-		if (fd > _max_fd)
-			_max_fd = fd;
-		std::cout << ""
-				  << ":" << it->get("port").get<double>() << " set up" << std::endl;
+		try
+		{
+			Socket *new_socket = new Socket(AF_INET, it->get("port").get<double>(), SOCK_STREAM, 0, *it);
+			int		fd = new_socket->get_socket_id();
+			FD_SET(fd, &_master_fd_set);
+			_sockets.insert(std::make_pair(fd, new_socket));
+			if (fd > _max_fd)
+				_max_fd = fd;
+			std::cout << it->get("hostname").get<std::string>() << ":" << it->get("port").get<double>()
+					  << std::endl;
+		}
+		catch (std::runtime_error)
```

Comment:
does that compile with `runtime` ?
```suggestion
		catch (std::exception & e)
```